### DISCUSSION TITLE
Support for <imagemap> elements

### DISCUSSION THREAD

**Discussion by osnard** [mpdf/mpdf#1915] [Q&A]
First of all: This is an awesome library! Thanks so much for this brilliant work.

I was wondering if there was some support for image maps like this?

```html
<img src="somefile.svg" alt="Workplace" usemap="#themap" />
<map name="themap">
  <area shape="rect" coords="50,50,250,400" href="https://somesite...">
</map> 
```

I couldn't find any hint neither in the docs, nor in the code. If you can give me a hint about how to implement this, I may be able to provide a patch.

**Comment by finwe** [comment]
I guess not likely, unless you find and sponsor a developer (or yourself) to explore, research and implement this. I don't even know if PDF standard itself can support this functionality.

**Comment by osnard** [comment]
Couldn't this be done using `LinkAnnotation`s that are placed freely at certain coordinates? I may be able to provide a patch myself, it'd just be nice to get a hint where to start.
